app/shared/components/Tools/CreateAccount.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+
+import { Header, Segment, Divider, Grid } from 'semantic-ui-react';
+
+import ToolsButtonCreateAccount from './Button/CreateAccount';
+import WalletPanelLocked from '../Wallet/Panel/Locked';
+
+class ToolsCreateAccount extends Component<Props> {
+  render() {
+    const {
+      accounts,
+      actions,
+      balances,
+      blockExplorers,
+      globals,
+      keys,
+      settings,
+      system,
+      validate,
+      wallet,
+      t
+    } = this.props;
+
+    const account = accounts[settings.account];
+
+    return (
+      <Grid centered>
+        <Grid.Column width={8} style={{ textAlign: 'center' }} >
```

Comment:
Instead of instantly using the `style` parameter for things like this, check the documentation:

https://react.semantic-ui.com/collections/grid

This should be:

```
<Grid.Column width={8} textAlign="center">
```

Reply:
thanks, didn't know about this!